tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "rootDir": ".",
     "skipLibCheck": true,
     "strict": true,
+    "strictNullChecks": true,
```

Comment:
This is redundant. `strict` already includes this flag

Reply:
Done